tests/psalm-baseline.xml

```diff
@@ -88,74 +82,15 @@
       <code>Command</code>
     </UndefinedClass>
   </file>
-  <file src="lib/Db/AliasMapper.php">
```

Comment:
I would prefer if only relevant changes are made to this file. Could you revert and `git add -p` the bits that need to be changed and leave the rest? If there are some leftovers then we should remove them in a clean, separate PR